docs/book/src/cronjob-tutorial/testdata/project/internal/controller/cronjob_controller.go

```diff
@@ -217,9 +217,7 @@ func (r *CronJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			continue
 		}
 		if scheduledTimeForJob != nil {
-			if mostRecentTime == nil {
-				mostRecentTime = scheduledTimeForJob
-			} else if mostRecentTime.Before(*scheduledTimeForJob) {
+			if mostRecentTime == nil || mostRecentTime.Before(*scheduldTimeForJob) {
```

Comment:
Great
But the change must be done in: https://github.com/kubernetes-sigs/kubebuilder/blob/c172069e83d81cc4a6ce67a7a1c3fd2a6e7e2d00/hack/docs/internal/cronjob-tutorial/controller_implementation.go#L208

Then, you will run make generate which will call the docs generate and it will get updated.

Reply:
Also, the emoji in this case is `:book:`

Reply:
Thanks, @camilamacedo86! Updated.